xml/System.ServiceModel.Federation/WSTrustChannel.xml

```diff
@@ -1,602 +1,602 @@
+      <MemberType>Method</MemberType>
+      <Implements>
+        <InterfaceMember>M:System.ServiceModel.Federation.IWSTrustChannelContract.IssueAsync(Microsoft.IdentityModel.Protocols.WsTrust.WsTrustRequest)</InterfaceMember>
+      </Implements>
+      <AssemblyInfo>
+        <AssemblyName>System.ServiceModel.Federation</AssemblyName>
+        <AssemblyVersion>6.0.0.0</AssemblyVersion>
+      </AssemblyInfo>
+      <ReturnValue>
+        <ReturnType>System.Threading.Tasks.Task&lt;System.IdentityModel.Tokens.SecurityToken&gt;</ReturnType>
+      </ReturnValue>
+      <Parameters>
+        <Parameter Name="trustRequest" Type="Microsoft.IdentityModel.Protocols.WsTrust.WsTrustRequest" />
+      </Parameters>
+      <Docs>
+        <param name="trustRequest">To be added.</param>
+        <summary>To be added.</summary>
+        <returns>To be added.</returns>
+        <remarks>To be added.</remarks>
+      </Docs>
+    </Member>
+    <Member MemberName="System.ServiceModel.ICommunicationObject.Abort">
+      <MemberSignature Language="C#" Value="void ICommunicationObject.Abort ();" />
+      <MemberSignature Language="ILAsm" Value=".method hidebysig newslot virtual instance void System.ServiceModel.ICommunicationObject.Abort() cil managed" />
+      <MemberSignature Language="DocId" Value="M:System.ServiceModel.Federation.WSTrustChannel.System#ServiceModel#ICommunicationObject#Abort" />
+      <MemberSignature Language="VB.NET" Value="Sub Abort () Implements ICommunicationObject.Abort" />
+      <MemberSignature Language="F#" Value="abstract member System.ServiceModel.ICommunicationObject.Abort : unit -&gt; unit&#xA;override this.System.ServiceModel.ICommunicationObject.Abort : unit -&gt; unit" Usage="wSTrustChannel.System.ServiceModel.ICommunicationObject.Abort " />
+      <MemberSignature Language="C++ CLI" Value=" virtual void System.ServiceModel.ICommunicationObject.Abort() = System::ServiceModel::ICommunicationObject::Abort;" />
+      <MemberType>Method</MemberType>
+      <Implements>
+        <InterfaceMember>M:System.ServiceModel.ICommunicationObject.Abort</InterfaceMember>
+      </Implements>
+      <AssemblyInfo>
+        <AssemblyName>System.ServiceModel.Federation</AssemblyName>
+        <AssemblyVersion>6.0.0.0</AssemblyVersion>
+      </AssemblyInfo>
+      <ReturnValue>
+        <ReturnType>System.Void</ReturnType>
+      </ReturnValue>
+      <Parameters />
+      <Docs>
+        <summary>Causes a communication object to transition immediately from its current state into the closed state.</summary>
+        <remarks>This member is an explicit interface member implementation. It can be used only when the &lt;xref:System.ServiceModel.Federation.WSTrustChannel&gt; instance is cast to an &lt;xref:System.ServiceModel.ICommunicationObject&gt; interface.</remarks>

```

Comment:
```suggestion
        <remarks>This member is an explicit interface member implementation. It can be used only when the <see cref="T:System.ServiceModel.Federation.WSTrustChannel"/> instance is cast to an <see cref="T:System.ServiceModel.ICommunicationObject"/> interface.</remarks>

```